lib/widgets/banner/banner.dart

```diff
@@ -31,44 +31,44 @@ class FlexBanner extends StatelessWidget {
 
   final Widget title;
 
-  final FlexImage image;
+  final Widget image;
 
   final Axis orientation;
 
   final FlexBannerImageLayout imageLayout;
 
   final CardTheme? theme;
 
-  // TODO Implement this, I'm thinking a heading line above the title to emphasise the title
   final Widget? heading;
```

Comment:
Nitpicky (maybe) again, but can we name this `overline` if that's what it is? It's the layout design industry term I believe.